### PR TITLE
React SDK & Nextjs: Support search and tags.

### PR DESCRIPTION
**Pull request by kevinstubbs**
- Fix pantheon.com => pantheon.io links
- Use "tags" instead of "keywords"
- Implement getAllTags
- Nextjs starter: support search bar and tags

### CODE REVIEW

File: nextjs-starter/components/tags.jsx [hunk 1]
```diff
@@ -0,0 +1,24 @@
+export const Tags = ({ tags }) => {
+  return (
+    <p className="mt-2 text-sm leading-[1.25rem] text-theme-bg-black">
+      {/* {tags.length ? (
```
Comment (a11rew) [inline nextjs-starter/components/tags.jsx:4]: Could we remove this component completely? Otherwise a comment giving context would be great

**Review by a11rew — CHANGES_REQUESTED** [review]

File: pcc/package.json [hunk 2]
```diff
@@ -1,5 +1,5 @@
 {
-  "name": "@pcc/starter-kit",
+  "name": "@pantheon-systems/pcc",
```
Comment (Aumkar) [inline pcc/package.json:2]: If app name is `pcc` then probably we can remove `bin` entry from this file

File: nextjs-starter/components/tags.jsx (hunk 1 above)
Comment (kevinstubbs) [reply nextjs-starter/components/tags.jsx:4]: Accidentally kept this commented out when I was debugging something. I've uncommented it. Good catch

File: pcc/package.json (hunk 2 above)
Comment (kevinstubbs) [reply pcc/package.json:2]: Hm, you're right, but also seems harmless to keep it.

**Pull request merged**